### DISCUSSION TITLE
Bug: Edit profile button broken when viewing starred list

### DISCUSSION THREAD

**Discussion by Christopher-Hayes** [community/community#9588] [Other Feature Feedback, Questions, & Ideas] [answered]
If you view a :star: list you created, and then try to use the "Edit Profile" button on the left-side of the screen, it doesn't do anything.

**Expected:** Pressing "Edit Profile" changes the UI to let you edit your profile description, etc.

**Actual:** Pressing "Edit Profile" on that page doesn't change the UI at all.

**Debugging:**
No errors show in console.
Reproduced bug on different lists.
Reproduced on Firefox 96 and Chrome 96 on Ubuntu 20.04.

https://user-images.githubusercontent.com/6013871/147950728-0127de42-e69a-4cea-833e-895fecfee235.mp4

**Comment by brianlovin** [comment, marked as answer]
I can reproduce this, thanks for the heads up @Christopher-Hayes — I've passed this along to our team!
